### DISCUSSION TITLE
歌词界面行间距很小/可以关闭逐字歌词吗

### DISCUSSION THREAD

**Discussion by lamberve** [amll-dev/applemusic-like-lyrics#234] [Q&A]
请问大佬们，我的歌词界面对比其他人行间距很小，如图，有解决办法吗？谢谢
另外还想请教一下逐字歌词动画可以关闭吗，谢谢！
![Snipaste_2024-02-18_11-27-56](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/31882886/f67e1616-1199-4dcb-a623-5ab63e75c419)

**Comment by lamberve** [comment]
AMLL版本是3.03

**Comment by lamberve** [comment]
网易云版本是2.10.12

**Comment by Linho1219** [comment]
AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

现关闭此讨论，敬请谅解。
